src/open_inwoner/configurations/management/commands/generate_config_docs.py

```diff
@@ -0,0 +1,115 @@
+import os
+from pathlib import Path
+from typing import TypeAlias
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+
+from open_inwoner.configurations.bootstrap.models import (
+    DigiDOIDCConfigurationSettings,
+    KICConfigurationSettings,
+    SiteConfigurationSettings,
+    ZGWConfigurationSettings,
+    eHerkenningDOIDCConfigurationSettings,
+)
+
+ConfigSetting: TypeAlias = (
+    DigiDOIDCConfigurationSettings
+    | KICConfigurationSettings
+    | SiteConfigurationSettings
+    | ZGWConfigurationSettings
+    | eHerkenningDOIDCConfigurationSettings
+)
+
+
+SUPPORTED_OPTIONS = ["siteconfig", "kic", "zgw", "digid_oidc", "eherkenning_oidc"]
```

Comment:
I'm seeing the same list quite a few times here, is there a way to make this simpler such that we don't have to add a new configuration in multiple places?